### PR TITLE
Feature/hmm export testset

### PR DESCRIPTION
**Pull request by ChataingT**
In regard to the ISSUE #14.
Here is a proposition to solve it.
The solution take part in the "betman segment_motifs" feature : 
- A boolean to save the hmm model
- An input path for a testing dataset that will be used only in hmm.predict()

### CODE REVIEW

**Comment by gchindemi** [comment]
Thank you for the contribution! I think the proposed solution can be further improved.
First, there is a minor bug: test_predictions is computed but not returned.
Second, the logic could be adapted to better support labeling the test set using a pre-trained model, as proposed in #14. To achieve this, I suggest modifying the function signature to allow the user to specify the path to a pre-trained model.
Additionally, it would be important to clearly warn users that any HMM configuration parameters (e.g., num_states) will be ignored when loading a pre-trained model. A log message should be more than enough.
